packages/frontend/amp/server/document.tsx

```diff
@@ -38,8 +45,12 @@ export const document = ({
 <html ⚡>
     <head>
     <meta charset="utf-8">
+
+    <!-- SEO related meta -->
```

Comment:
I guess if you're going to signpost here, you could signpost when you come out of here (or just remove this one 😄 )